File: sentimental-cash/cash.py
from cs50 import get_float

while True:
    cash = get_float("Change: ")
    if (cash > 0):
        break

count = 0
cash = int(cash * 100)

while cash > 0:
    if (cash - 25 >= 0):
        count += 1
        cash = cash - 25
    elif (cash - 10 >= 0):
        count += 1
        cash = cash - 10
    elif (cash - 5 >= 0):
        count += 1
        cash = cash - 5
    elif (cash - 1 >= 0):
        count += 1
        cash = cash - 1
print(count)

File: cash/cash.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int change;
    do
    {
        change = get_int("Change: ");
    }
    while (change <= 0);

    int count = 0;

    do
    {
        if (change - 25 >= 0)
        {
            count++;
            change -= 25;
        }
        else if (change - 10 >= 0)
        {
            count++;
            change -= 10;
        }
        else if (change - 5 >= 0)
        {
            count++;
            change -= 5;
        }
        else if (change - 1 >= 0)
        {
            count++;
            change -= 1;
        }
    }
    while (change > 0);

    printf("%i\n", count);
}
